src/pages/instances/MigrateInstanceModal.tsx

```diff
@@ -1,197 +1,100 @@
-        status: member.status.toLowerCase(),
-      },
-    };
-  });
-
-  if (isLoading) {
-    return <Loader />;
+  let modalTitle: ReactNode = isClustered
+    ? `Choose migration method for instance ${instance.name}`
+    : `Choose the target storage pool for instance ${instance.name}`;
+
+  const migrationEntity = isClustered
+    ? type?.split("-").join(" ")
+    : "storage pool";
+
+  if (["cluster-member", "storage-pool"].includes(type || "")) {
+    modalTitle = (
+      <BackLink
+        title={`Choose the target ${migrationEntity} for instance ${instance.name}`}
+        onClick={() => setType(null)}
+        linkText="Choose migration method"
+      />
+    );
   }
 
-  const modalTitle = selectedMember
-    ? "Confirm migration"
-    : `Choose target cluster member for instance ${instance.name}`;
-
-  const summary = (
-    <div className="migrate-instance-summary">
-      <p>
-        This will migrate instance <strong>{instance.name}</strong> to cluster
-        member <b>{selectedMember}</b>.
-      </p>
-    </div>
-  );
+  if (target) {
+    modalTitle = (
+      <BackLink
+        title="Confirm migration"
+        onClick={() => setTarget("")}
+        linkText={`Choose ${migrationEntity}`}
+      />
+    );
+  }
 
   return (
     <Modal
       close={close}
       className="migrate-instance-modal"
       title={modalTitle}
-      buttonRow={
-        <div id="migrate-instance-actions">
-          <Button
-            className="u-no-margin--bottom"
-            type="button"
-            aria-label="cancel migrate"
-            appearance="base"
-            onClick={handleCancel}
-          >
-            Cancel
-          </Button>
-          <ActionButton
-            appearance="positive"
-            className="u-no-margin--bottom"
-            onClick={() => handleMigrate(instance)}
-            disabled={!selectedMember}
-          >
-            Migrate
-          </ActionButton>
-        </div>
-      }
       onKeyDown={handleEscKey}
     >
-      {selectedMember ? (
-        summary
-      ) : (
-        <div className="migrate-instance-table u-selectable-table-rows">
-          <ScrollableTable
-            dependencies={[]}
-            tableId="migrate-instance-table"
-            belowIds={["status-bar", "migrate-instance-actions"]}
-          >
-            <MainTable
-              id="migrate-instance-table"
-              headers={headers}
-              rows={rows}
-              sortable
-              className="u-table-layout--auto"
-            />
-          </ScrollableTable>
+      {isClustered && !type && (
+        <div className="choose-migration-type">
+          <FormLink
+            icon="machines"
+            title="Migrate instance to a different cluster member"
+            onClick={() => setType("cluster-member")}
+          />
+          <FormLink
+            icon="pods"
+            title="Migrate instance to a different storage pool"
```

Comment:
```suggestion
            title="Migrate instance root storage to a different pool"
```

Reply:
I double check the modal heading, summary copy and notification copy. It's consistently referring to "root storage" now.